src/main/webapp/js/utils/form/form-utils.jsx

```diff
@@ -115,15 +115,23 @@ const renderFormField = (props) => {
   );
 };
 
-export const validate = fieldConfig => (values) => {
+export const validate = (fieldConfig, entities, entityToEdit) => (values) => {
   const errors = {};
-
   _.forEach(fieldConfig, (config) => {
     const val = values[config.name];
 
     if (config.required && (_.isNil(val) || val === '')) {
       errors[config.name] = `${config.displayName} is required`;
     }
+    if (config.uniqueField) {
```

Comment:
you can check if the value is not nil and not empty here and only then use the find